src/test/java/dk/cngroup/lentils/service/FinalPlaceServiceIntegrationTest.java

```diff
@@ -47,10 +49,10 @@ public void testSaveFinalPlaceOverExistingPlace() {
         finalPlaceRepository.save(originalPlace);
         assertEquals(1, finalPlaceRepository.findAll().size());
 
-        final String placeDescription = "ABCDE";
+        final String placeDescription = TEST_DESCRIPTION;
```

Comment:
Tady v těch testech by se to nedalo zkrátit a použít ten ObjectGenerátor?
Vytvořit třeba validní entiu přes něj a pak jen nastavit tu konkrétní proměnou co potřebuješ pro test?

Reply:
upravil jsem to trosku, prosim o kontrolu, jestli jsem te dobre pochopil

Reply:
Jo, pecka. Může být 👍